forte/processors/base/base_processor.py

```diff
@@ -38,7 +38,6 @@ def __init__(self):
         super().__init__()
         self.selector = DummySelector()
 
-    @abstractmethod
     def new_pack(self, pack_name: Optional[str] = None) -> PackType:
```

Comment:
You cannot simply remove the keyword, you also need to replace "NotImplemented" with "pass" or something like that.